assets/js/components/HostSettings/HostSettings.test.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+
+import { screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+
+import { faker } from '@faker-js/faker';
+import {
+  withState,
+  defaultInitialState,
+  renderWithRouterMatch,
+} from '@lib/test-utils';
+import { catalogCheckFactory, hostFactory } from '@lib/test-utils/factories';
+
+import HostSettings from './HostSettings';
+
+describe('HostDetails HostSettings component', () => {
+  it('should render the host info box and the catalog container', async () => {
+    const group0 = faker.animal.cat();
+    const group1 = faker.animal.dog();
+    const group2 = faker.lorem.word();
+    const catalog = [
+      ...catalogCheckFactory.buildList(2, { group: group0 }),
+      ...catalogCheckFactory.buildList(2, { group: group1 }),
+      ...catalogCheckFactory.buildList(2, { group: group2 }),
+    ];
+    const hosts = hostFactory.buildList(3, { provider: 'azure' });
+
+    const state = {
+      ...defaultInitialState,
+      catalog: { loading: false, data: catalog, error: null },
+      hostsList: { hosts },
+    };
+    const { id: hostID, agent_version: agentVersion } = hosts[1];
+
+    const [StatefulChecksSettings] = withState(<HostSettings />, state);
+
+    renderWithRouterMatch(StatefulChecksSettings, {
+      path: 'hosts/:hostID/settings',
+      route: `/hosts/${hostID}/settings`,
+    });
+
+    expect(screen.getByText('Provider')).toBeVisible();
+    expect(screen.getByText('Azure')).toBeVisible();
+    expect(screen.getByText('Agent version')).toBeVisible();
+    expect(screen.getByText(agentVersion)).toBeVisible();
+    expect(screen.getByText(group0)).toBeVisible();
+    expect(screen.getByText(group1)).toBeVisible();
+    expect(screen.getByText(group2)).toBeVisible();
+
```

Comment:
What about making the test a bit stronger and adding a click event to check if the created groups also have the checks in them ? :D

Reply:
I see, however that's already tested in the `assets/js/components/ChecksSelection/ChecksSelectionGroup.test.jsx`.
That should be enough. What do you think?

Reply:
Aye this works!